include/nuttx/wqueue.h

```diff
@@ -356,8 +359,17 @@ int work_usrstart(void);
  *
  ****************************************************************************/
 
-int work_queue(int qid, FAR struct work_s *work, worker_t worker,
-               FAR void *arg, clock_t delay);
+#if defined(CONFIG_LIB_USRWORK) && !defined(__KERNEL__)
+  int work_queue(int qid, FAR struct work_s *work, worker_t worker,
+                 FAR void *arg, clock_t delay);
+#else
+  int work_queue_nt(int qid, FAR struct work_s *work, worker_t worker,
+                    FAR void *arg, clock_t delay, FAR const char *name_tag);
+
+/* Stringificate the worker name for debug purposes */
+
+#  define work_queue(qid, work, worker, arg, delay) work_queue_nt(qid, work, worker, arg, delay, #worker)
```

Comment:
```suggestion
#  define work_queue(qid, work, worker, arg, delay) \
                     work_queue_nt(qid, work, worker, arg, delay, #worker)
```
Line break to no exceed the 78 chars line limit.

Reply:
Good. Thanks.